File: app/src/main/java/com/itn/securebrowser/util/LocaleHelper.java
package com.itn.securebrowser.util;

import android.content.Context;
import android.content.res.Configuration;
import java.util.Locale;

public class LocaleHelper {

    public static final String APP_LOCALE = "en";

    public static Context wrap(Context context) {
        Locale locale = new Locale(APP_LOCALE);
        Locale.setDefault(locale);
        Configuration config = new Configuration(context.getResources().getConfiguration());
        config.setLocale(locale);
        return context.createConfigurationContext(config);
    }
}

File: app/src/main/java/com/itn/securebrowser/LocaleHelper.kt
package com.itn.securebrowser

import android.content.Context
import android.content.res.Configuration
import java.util.Locale

object LocaleHelper {

    /**
     * Change this constant to switch the app language.
     * Examples: "en" for English, "ar" for Arabic, "fr" for French.
     */
    const val APP_LOCALE = "en"

    fun wrap(context: Context): Context {
        val locale = Locale(APP_LOCALE)
        Locale.setDefault(locale)
        val config = Configuration(context.resources.configuration)
        config.setLocale(locale)
        return context.createConfigurationContext(config)
    }
}
